test/conftest.py

```diff
@@ -12,37 +12,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Dict
+from typing import Any, Dict, List, TypeAlias
 
 import pytest
 
+_ConfSingleValue: TypeAlias = bool | int | float
+ConfigValue: TypeAlias = _ConfSingleValue | List[_ConfSingleValue] | Dict[str, _ConfSingleValue]
+
 
 @pytest.fixture
-def fast_config() -> Dict[str, Dict[str, bool | int | float]]:
+def fast_config_modifications() -> Dict[str, ConfigValue]:
     return {
-        "system": {
-            # common
-            "num_updates": 2,
-            "rollout_length": 1,
-            "num_minibatches": 1,
-            "update_batch_size": 1,
-            # ppo:
-            "ppo_epochs": 1,
-            # sac:
-            "explore_steps": 1,
-            "epochs": 1,  # also for iql
-            "policy_update_delay": 1,
-            "buffer_size": 8,  # also for iql
-            "batch_size": 1,
-            # iql
-            "min_buffer_size": 4,
-            "sample_batch_size": 1,
-            "sample_sequence_length": 1,
-        },
-        "arch": {
-            "num_envs": 1,
-            "num_eval_episodes": 1,
-            "num_evaluation": 1,
-            "absolute_metric": False,
-        },
+        # ---------- system config ---------
+        # common
+        "num_updates": 2,
+        "rollout_length": 1,
+        "num_minibatches": 1,
+        "update_batch_size": 1,
+        # ppo
+        "ppo_epochs": 1,
+        # sac
+        "explore_steps": 1,
+        "epochs": 1,  # also for iql
+        "policy_update_delay": 1,
+        "buffer_size": 8,  # also for iql
+        "batch_size": 1,
+        # iql
+        "min_buffer_size": 4,
+        "sample_batch_size": 1,
+        "sample_sequence_length": 1,
```

Comment:
Just so it plays nice with `rec_iql` in the tests. 

```suggestion
        "sample_sequence_length": 2,
```